### DISCUSSION TITLE
impossibile aggiornare firmware per ortus laser master 2

### DISCUSSION THREAD

**Discussion by sylunder** [arkypita/LaserGRBL#1604] [Q&A]
Buongiorno a tutti, io non riesco ad aggiornare il firmware del mio Ortur Laser Master 2 pur seguendo le indicazioni.
Avete una soluzione?

**Comment by arkypita** [comment]
Quando ho aggiornato il mio non ho avuto problemi, ho semplicemente seguito alla lettera le indicazioni che si trovano nel file zip che scarichi da qui: https://ortur.tech/firmware/OLM_Series/OLM_2_Firmware_Release_1.4.zip

Ad ogni modo questo non è il posto giusto dove chiedere aiuto, troveresti sicuramente persone più competenti nella [pagina facebook di Ortur](https://www.facebook.com/groups/orturusergroup) oppure contattandoli al [loro servizio assistenza](https://ortur.tech/support/).

Esistono anche dei gruppi facebook formati da utenti italiani:
https://www.facebook.com/ORTUR-LASER-Italia-105508624645468/
https://www.facebook.com/groups/orturlasermaster2proita/
